gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/ExpandableText.kt

```diff
@@ -37,25 +46,53 @@ internal fun ExpandableText(text: String, modifier: Modifier = Modifier, maxLine
     )
     if (showDialog) {
         Dialog(onDismissRequest = { showDialog = false }) {
-            Card(
-                modifier = Modifier.wrapContentSize(),
-                shape = RoundedCornerShape(16.dp),
-            ) {
-                Text(
-                    text = text,
-                    modifier = Modifier
-                        .padding(16.dp)
-                        .fillMaxWidth()
-                        .wrapContentSize(),
-                    textAlign = TextAlign.Start,
-                )
+            if (dialogContent != null) {
+                dialogContent()
```

Comment:
I know we can write something like:
```
    val text = "Text that \ncan be expanded \nby clicking on it."
    ExpandableText(text) {
        Card { Text(text) }
    }
```
But it seems more natural to me to change the composable signature to `((String) -> Unit)?` and pass the `text` here: `dialogContent(text)`? I would expect the "full string" to be displayed somewhere, and so we could just write this:

```
    ExpandableText("Text that \ncan be expanded \nby clicking on it.") {
        Card { Text(it) }
    }
```

Reply:
Sounds good! I've updated the code. Let me know what you think.